src/components/content/common/ocl/DeploymentText.tsx

```diff
@@ -15,15 +15,15 @@ export function DeploymentText({ deployment }: { deployment: Deployment }): Reac
         const yamlDocument = new YAML.Document();
         // eslint-disable-next-line @typescript-eslint/ban-ts-comment
         // @ts-expect-error
-        yamlDocument.contents = deployment;
+        yamlDocument.contents = deployment.serviceAvailabilityConfigs;
         return (
             <Popover
                 content={<pre className={oclDisplayStyles.oclDeploymentScript}>{yamlDocument.toString()}</pre>}
-                title={'Deployment'}
+                title={'Service Availability Configs'}
                 trigger='hover'
             >
                 <Button className={oclDisplayStyles.oclDataHover} type={'link'}>
-                    {deployment.deployerTool.kind}
+                    {`configs`}
```

Comment:
rename to 'service availability'

Reply:
![image](https://github.com/user-attachments/assets/42dde3f3-752b-45a1-a74f-b2f826bbb129)

fixed.

Reply:
![image](https://github.com/user-attachments/assets/fe4fd764-f83f-4f0e-b7a4-60b3ef4cc622)

![image](https://github.com/user-attachments/assets/2ba23833-6729-447a-9d19-53c9339f7b53)
fixed.